app/constants/general.ts

```diff
@@ -21,6 +21,9 @@ export default {
     OPEN_CHANNEL: 'O' as const,
     PRIVATE_CHANNEL: 'P' as const,
     GM_CHANNEL: 'G' as const,
+    MANAGE_MEMBERS_OPTIONS: {
+        REMOVE_USER: 'remove',
+    },
```

Comment:
does this constant really belong in general?

Reply:
created members.ts and moved into this file.